xtext/org.icyphy.linguafranca/src/org/icyphy/generator/GeneratorBase.xtend

```diff
@@ -137,6 +145,11 @@ abstract class GeneratorBase extends AbstractLinguaFrancaValidator {
      */
     protected var iResource = null as IResource
     
+    /** 
```

Comment:
To my knowledge, `ReactorInstance` is only used in the C code generator and doesn't belong in `GeneratorBase`. If this changed, what is the TypeScript code generator now using `ReactorInstance` for? This class is meant to unfold the entire instantiation hierarchy and carry out analysis that the C++ and TS target perform at runtime.

Reply:
When I refactored the RTI generation functions out of CGenerator and into GeneratorBase, I performed a pretty mechanical operation of moving all of the variables used for RTI generation into GeneratorBase. `main` is currently used on line 826 of the function `compileCommand` in GeneratorBase. The TypeScript generator invokes compileCommand.